.agents/skills/webawesome/references/components/toast.md

```
@@ -0,0 +1,509 @@
+
+Toasts have a number of accessibility limitations. For example:
+
+-   Due to their transient nature, toasts can be missed entirely by screen magnifier users and difficult to perceive for others depending on their visual and cognitive abilities.
+-   Due to their DOM placement and vague position in a page's reading order, toasts can be difficult for keyboard users to navigate to.
+-   Due to their positioning on the page, toasts can obscure other essential elements, especially for users who use browser or OS zooming features.
+
+Rarely will toasts offer the best usability for all of your users, so consider if other UX patterns, like dialogs or inline messages, offer a better experience. If toasts are the right fit for your use case, consider these tips:
+
+-   Keep toasts short and sweet.
+-   Set the `duration` to `5000` ms (5 seconds) or longer to give users enough time to locate and understand the toast item.
+-   Consider allowing users to adjust the timing. While the duration of a toast item resets on hover (see [Hover & Focus Behavior](#hover-and-focus-behavior)), a setting in your application can allow users to control the timing of toasts to best suit their needs and abilities.
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)